components/AvalancheForecastZoneMap.tsx

```diff
@@ -92,13 +102,112 @@ export const CARD_WHITESPACE = (1 - CARD_WIDTH) / 2; // proportion of overall wi
 export const CARD_SPACING = CARD_WHITESPACE / 2; // proportion of overall width that the spacing between two cards takes up
 export const CARD_MARGIN = CARD_SPACING / 2; // proportion of overall width that each card needs as a margin
 
+enum AnimatedDrawerState {
+  Hidden = 'Hidden',
```

Comment:
Hidden state is currently unused, but will eventually be the default - then tapping a region in the map will make the zone cards visible. See Figma mocks for more details.

This PR just focuses on the mechanics of making the zone cards (the drawer) able to be pulled up & down.